package.json

```diff
@@ -3,12 +3,12 @@
   "version": "4.0.28-development",
   "description": "Web Experience Toolkit (WET): Open source code library for building innovative websites that are accessible, usable, interoperable, mobile-friendly and multilingual. This collaborative open source project is led by the Government of Canada. / Boîte à outils de l’expérience Web (BOEW) : Une bibliothèque de code primée pour construire des sites Web innovants qui sont accessibles, faciles d'emploi, interopérables, optimisés pour les appareils mobiles et multilingues. Ceci est un projet à source ouverte collaboratif dirigé par le Gouvernement du Canada.",
   "main": "index.html",
-  "engines" : { 
-    "npm" : "~4.0.0" 
+  "engines" : {
+    "npm" : "~4.0.0"
```

Comment:
`>=4.0.0` so people running 5+ on Node 8 don't get warnings

Reply:
I don't get warning and I am running npm 5